packages/logging/src/metric-logger.ts

```diff
@@ -0,0 +1,36 @@
+import { MetricLoggerOptions } from './types';
+
+const DefaultConfig = {
+  url: `https://api.datadoghq.com/api/v2/series`,
+};
+
+let config: MetricLoggerOptions;
+
+export function initMetricLogger(opts: MetricLoggerOptions) {
+  const { apiKey, url } = opts;
+  config = {
+    ...DefaultConfig,
+    url,
+    apiKey,
+  };
+}
+
+export async function submitMetrics(series: DDSeries) {
+  try {
+    const body = JSON.stringify({ series });
+    const opts = {
+      method: 'POST',
+      body,
+      headers: {
+        'content-type': 'application/json',
+        'dd-api-key': config.apiKey,
+      },
+    };
+
+    const resp = await fetch(config.url ?? DefaultConfig.url, opts);
+    // console.log(`submitMetrics: ${JSON.stringify(resp)}`);
+    return resp;
+  } catch (e) {
+    console.error(e);
```

Comment:
If the fetch request errors out, this will swallow the error and no metric will be submitted. Given how we're logging errors in the workers we won't be alerted to issues.

I think this should either log to datadog or allow the error to be thrown.

Reply:
It now throws the error, I will clean up the endpoints along with implementing the api.